x-pack/plugins/monitoring/public/components/apm/overview/status.js

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
Could this file could be added as .ts?

Reply:
Yeah, good point. I don't want to add to the debt 😅

Reply:
Okay, I take that back. So the problem is `import { Status as InstancesStatus } from '../instances/status';` imports something that hasn't been converted to TS, which leads to errors. I took a look at the code in there and it's not super quick to convert over. 

I'll leave this for now as the damage footprint is pretty low (it just imports another component). 

This will get picked up by future conversion tickets for SM.